pkg/operation/botanist/controlplane.go

```diff
@@ -116,6 +116,32 @@ func (b *Botanist) deleteNamespace(ctx context.Context, name string) error {
 	return err
 }
 
+// EnsureClusterIdentity ensures that Shoot cluster-identity ConfigMap exists and stores its data
+// in the operation. Updates shoot.status.clusterIdentity if it doesn't exist already.
+func (b *Botanist) EnsureClusterIdentity(ctx context.Context) error {
+	return b.Shoot.Components.ClusterIdentity.Deploy(ctx)
+}
+
+// determineShootClusterIdentity determines the identity of a Shoot cluster, in cases that the identity exists it returns its value, else computes the identity and returns it.
+func (b *Botanist) determineShootClusterIdentity(ctx context.Context) (string, error) {
```

Comment:
This looks like the same code as in `pkg/operation/botanist/clusteridentity/clusteridentity.go`.

Reply:
It is a leftover, I will delete it